docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -543,98 +569,145 @@ paths:
           required: false
           type: string
         - name: tag
           in: query
+          example: "artificial intelligence"
           description: |
             Limit the results to annotations tagged with the specified value.
 
             For example: `artificial intelligence` will find all annotations whose tags
-            contain both artificial and intelligence.
+            contain both `artificial` **AND** `intelligence`.
           required: false
           type: string
         - name: tags
           in: query
+          example:
+            - artificial
+            - intelligence
           description: |
-            Alias of `tag` but allows a comma separated list specification for multiple tags.
+            Similar to `tag` but allows a comma-separated list of multiple tags.
 
             For example: `[intelligence,artificial]` will find all annotations whose tags
-            contain both artificial and intelligence.
+            contain both `artificial` **AND** `intelligence`.
           required: false
           type: string
         - name: any
           in: query
-          description: Limit the results to annotations whose quote, tags, text or url
-            fields contain this keyword.
+          example: "ribosome"
+          description: |
+            Limit the results to annotations who contain the indicated keyword
+            in any of the following fields:
+
+            * `quote`
+            * `tags`
+            * `text`
+            * `url`
           required: false
           type: string
         - name: quote
           in: query
-          description: Limit the results to annotations that contain this text inside
-            the text that was annotated.
+          example: "unicorn helmets"
+          description: >
+            <p>Limit the results to annotations that contain this text inside
+            the text that was annotated.</p>
+
+            <p>For example: `unicorn helmets` would return all annotations containing
+            `unicorn` **OR** `helmets` in their quoted (i.e. annotated) text.</p>
+
           required: false
           type: string
         - name: references
           in: query
-          description: Returns annotations that are replies to this parent annotation id.
+          description: Returns annotations that are replies to this parent annotation ID.
           required: false
           type: string
         - name: text
           in: query
-          description: Limit the results to annotations that contain this text in their
-            textual body.
+          example: "penguin strength"
+          description: >
+            <p>Limit the results to annotations that contain this text in their
+            textual body.</p>
+
+            <p>For example: `penguin strength` would return all annotations containing
+            `penguin` or `strength` in their text (body) content.</p>
```

Comment:
This is a nitpick but I noticed you used a bolded **OR** in one of the other params above so might be worth doing here as well?